contracts/src/v0.8/dev/KeeperRegistryLogic.sol

```diff
@@ -140,7 +140,9 @@ contract KeeperRegistryLogic is KeeperRegistryBase {
     uint32 gasLimit,
     address admin,
     bytes calldata checkData
-  ) external onlyOwnerOrRegistrar returns (uint256 id) {
+  ) external returns (uint256 id) {
+    if (msg.sender != owner() && msg.sender != s_registrar) revert OnlyCallableByOwnerOrRegistrar();
```

Comment:
I like this 10x more than using the modifier. so nice to have all the code in one place.